src/test/scala/dissonance/TestUtils.scala

```diff
@@ -5,11 +5,12 @@ import cats.effect.IO
 import cats.effect.ContextShift
 
 object TestUtils {
+  // TODO: Return Stream[IO, String] of lines from file
   def readFileFromResource(path: String)(implicit contextShift: ContextShift[IO]) = {
```

Comment:
We should probably add a return type here to be explicit imo

Reply:
Will do